README.md

```diff
@@ -1,4 +1,151 @@
+
+## Libraries / Frameworks
+
+Smart Contract SDK
+
+* [near-sdk-rs](https://github.com/near/near-sdk-rs)
+* [near-sdk-js](https://github.com/near/near-sdk-js)
+* [near-sdk-as](https://github.com/near/near-sdk-as)
+
+Frontend Libraries
+
+* [near-api-js](https://github.com/near/near-api-js)
+* [near-api-go](https://github.com/textileio/near-api-go)
+* [near-api-rs](https://github.com/bicarus-labs/near-api-rs)
+* [near-api-py](https://github.com/near/near-api-py)
+* [near-api-swift](https://github.com/near/near-api-swift)
+
+Testing Tools
+
+* [near-workspaces-rs](https://github.com/near/workspaces-rs)
+* [near-workspaces-js](https://github.com/near/workspaces-js)
+
+Game Frameworks
+
+* [unity](https://github.com/near/near-api-unity)
+* [godot](https://github.com/svntax/godot-near-sdk)
+
+Wallet SDK
+
+* [wallet-selector](https://github.com/near-projects/wallet-selector)
+
+## Tools
+
+* [NEAR Explorer](https://explorer.near.org/)
+* [NEAR Wallet](https://wallet.near.org)
+* [Developer Console](https://console.near.dev/)
+
+
+## Projects
+
+### Basics
+
+- https://examples.near.org/
+
+* [Guest book](https://github.com/near-examples/guest-book)
+* [Counter](https://github.com/near-examples/counter)
+
+### Infrastructure
+
+* [Octopus](https://github.com/octopus-network)
+* [Aurora](https://github.com/aurora-is-near)
+
+### FT
+
+* [FT example](https://github.com/near-examples/FT)
+* [wNEAR](https://github.com/near/core-contracts/blob/master/w-near)
+
+### DeFi
+
+* [Ref Finance](https://github.com/ref-finance/ref-contracts)
+* [Skyward](https://github.com/skyward-finance/)
+* [Meta Pool](https://github.com/Narwallets/meta-pool)
+* [OIN DAO](https://github.com/oinfinance/NearOinDao3.0)
+
+### NFT
+
+* [NFT example](https://github.com/near-examples/NFT)
+* [Paras](https://github.com/ParasHQ/paras-marketplace-contract)
+* [Mintbase](https://github.com/Mintbase/mintbase-core)
+* [TenK](https://github.com/TENK-DAO/tenk)
+
+### DAO
+
+* [SputnikDAO](https://github.com/near-daos/sputnik-dao-contract)
+
+### Oracle
+
+* [Flux](https://github.com/fluxprotocol)
+
+### Linkdrop
+
+* [NEAR Linkdrop](https://github.com/near/near-linkdrop)
+* [Linkrop](https://github.com/NEAR-labs/contracts.near-linkdrop)
+* [NFT Linkdrop](https://github.com/danielwpz/nft-linkdrop)
+
+### Game
+
+* [BerryClub](https://github.com/evgenykuzyakov/berryclub)
+* [NEAR Lands](https://github.com/vgrichina/near-lands)
+
+### Social
+
+* [web3.bio](https://github.com/picturepan2/web3bio)
+
+## Validators / RPCs
+
+* [Introduction](https://docs.near.org/docs/develop/node/intro/what-is-a-node)
+
+### RPC Nodes
+
+#### Non-Archival Nodes
+
+1. mainnet https://rpc.mainnet.near.org
+2. testnet https://rpc.testnet.near.org
+
+Alternatives
+
+1. https://near-drpc.blockpi.io/
+2. https://near-drpc.blockpi.io/testnet/
+
+
+#### Archival Nodes
+
+1. mainnet https://archival-rpc.mainnet.near.org
+2. testnet https://archival-rpc.testnet.near.org
+
+
+
+## Decetralized Storage
+
```

Comment:
Let's link the docs we created for some of the solutions: https://docs.near.org/docs/concepts/storage-solutions